react/components/loaders/FetchMoreButton.js

```diff
@@ -23,6 +24,28 @@ const useShowButton = (to, products, loading, recordsFiltered) => {
   return showButton
 }
 
+function shouldNotIncludeMap(map) {
```

Comment:
What do you think about reuse this function from the vtex.store?

Reply:
I considered it, but I thought it was too much hustle since `vtex.store` already depends on `vtex.search-result`. I'm afraid to deal with complicated problems that may arise from such a simple requirement.